source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ADCIOSAdaptiveHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ADCIOSAdaptiveHostConfig:NSObject
+
+- (CGSize) getImageSize:(std::shared_ptr<Image>) imgBlock;
+
+- (UIColor* ) getTextBlockColor:(std::shared_ptr<AdaptiveCards::TextBlock>) txtBlock;
+
+- (int) getTextBlockTexSize:(std::shared_ptr<TextBlock>)txtBlock;
+
+- (NSTextAlignment) getTextBlockAlignment:(std::shared_ptr<TextBlock>) txtBlock;
+
+- (NSNumber* ) getTextBlockTextWeight:(std::shared_ptr<TextBlock>) txtBlock;
```

Comment:
Both of these are part of the TextBlock element itself, they do not come from hostConfig. Maybe this is the wrong place for these?